### DISCUSSION TITLE
希望可以增加原文样式的控制

### DISCUSSION THREAD

**Discussion by murazi** [immersive-translate/immersive-translate#542] [Ideas]
### 新功能 | New Feature

1.对原文样式的控制

### 为什么您想要添加这些功能 | Why you want to add these features

通常我们经常访问的网站样式是我们比较熟悉的，所以会习惯性的去关注那些习惯了的内容，而对译文加强的效果会很别扭，既然是沉浸式翻译，我认为应该在保留原始格式和内容的基础上去优化翻译体验，而非反道其行

曾经在TG上提过这个需求，被很多人反驳过这个需求，我再提一次试试看吧：D

### 提议 | Proposal

尝试增加一个小选项，对译文进行处理，对原文进行处理

**Comment by versun** [comment]
不是很明白你的需求，能举个例子说明下吗

**Reply by murazi** [reply]
> 不是很明白你的需求，能举个例子说明下吗

![](https://obsidiancdnpack.luolala.com/obsidiancdnpack/202306/8579c19e4941de4323aecbdaec1af845.webp)

希望后台可以设置，是对译文进行样式处理，或是原文进行样式处理
因为我的需求是，对原文进行弱化样式处理，会更复合原本网站的阅读体验

**Reply by versun** [reply]
@theowenyoung @keyboard3 
两位大佬看看哈

**Comment by theowenyoung** [comment]
修改原文样式会破坏不少网站的功能，所以这个功能应该在可见的未来不会考虑。

译文是我们额外插入的内容，所以我们可以做较多的控制，但是对于原文内容我们的原则是尽量不改变原文的结构和状态，避免影响原网页的功能。

**Reply by murazi** [reply]
好的明白了,感谢答复
